runtime/bundles/org.eclipse.e4.core.di/META-INF/MANIFEST.MF

```diff
@@ -15,6 +15,8 @@ Export-Package: org.eclipse.e4.core.di;version="1.7.0",
 Require-Bundle: org.eclipse.e4.core.di.annotations;bundle-version="[1.4.0,2.0.0)";visibility:=reexport
 Import-Package: javax.annotation;version="[1.3.5,2.0.0)",
  javax.inject;version="[1.0.0,2.0.0)",
+ jakarta.inject;version="[2,3)",
+ jakarta.annotation;version="[2,3)",
```

Comment:
If the import of both is not optional, we always force both API bundles to be present in the runtime as long as both are supported. This would be sub-optimal.